ScoutSuite/output/data/html/partials/aws/left_menu_for_region.html

```diff
@@ -1,9 +1,9 @@
 <!-- Generic left-menu link partial for region-then-resource -->
 <script id="left_menu_for_region.partial" type="text/x-handlebars-template">
-  <div id="services.cloudtrail.regions.id.trails.list">
+  <div id="services.cloudtrail.regions.id.trails.list" />
```

Comment:
Why did this slash appear?

Reply:
I think it got removed but since this PR is in the past GitHub tells us this was removed in the future